src/shader.wgsl

```diff
@@ -32,6 +32,15 @@ var mask_atlas_texture: texture_2d<f32>;
 @group(0) @binding(3)
 var atlas_sampler: sampler;
 
+fn srgb_to_linear(srgb: u32) -> f32 {
```

Comment:
This could be improved slightly once more component-wise functions are available in naga, e.g., component-wise `mix` and `step` to calculate all components